week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(price) {
+  return "$".concat(price / 1.4);
```

Comment:
This is supposed to convert from pounds to dollars, so for example £10 will become $14 (the dollar value should be larger). If a function doesn't provide examples, then it's a good idea to think some up for yourself and check if they are correct :-). The code is all good though.